components/style/properties/gecko.mako.rs

```diff
@@ -1344,11 +1343,11 @@ fn static_assert() {
 <%def name="define_ffi_struct_accessor(style_struct)">
 #[no_mangle]
 #[allow(non_snake_case, unused_variables)]
-pub extern "C" fn Servo_GetStyle${style_struct.gecko_name}(computed_values: *mut bindings::ServoComputedValues)
+pub extern "C" fn Servo_GetStyle${style_struct.gecko_name}(computed_values: ServoComputedValuesBorrowed)
   -> *const ${style_struct.gecko_ffi_name} {
-    type Helpers = ArcHelpers<bindings::ServoComputedValues, ComputedValues>;
-    Helpers::with(computed_values, |values| values.get_${style_struct.name_lower}().get_gecko()
-                                                as *const ${style_struct.gecko_ffi_name})
+    let values = <ComputedValues as RefCounted>::from_borrowed(computed_values);
```

Comment:
Because these are bare functions, sometimes we have to hint to the type checker :/ Theoretically there could be two different types that have ServoComputedValuesBorrowed as their borrowed type, so the type checker isn't certain.

I wish there was a way of expressing proper 1-1 duality/triality in the type system, but there isn't.

Fortunately this isn't usually necessary, just in cases where the self type doesn't get specified anywhere nearby.